specifics/SpectralInstrument/SpectralInstrument.h

```diff
@@ -65,6 +65,12 @@ using namespace yat4tango;
 
 namespace Spectral_ns
```

Comment:
please rename the Tango Class from Spectral to SpectralInstrument: the namespace and the class name